LongWarOfTheChosen/Config/XComLW_FactionBalance.ini

```diff
@@ -86,11 +89,19 @@ OVERRIDE_REDUCTION=10
 
 [LW_FactionBalance.X2LWModTemplate_ReaperAbilities]
 SHADOW_DURATION=1
-SHADOW_DETECTION_RANGE_REDUCTION=0.7f
+SHADOW_DETECTION_RANGE_REDUCTION=0.9f
 
 REMOTE_START_CHARGES=1
 REMOTE_START_DEMOLITIONIST_CHARGES=1
 
+BLOOD_TRAIL_ANTIDODGE_BONUS=20
+
+PALE_HORSE_BASE_CRIT=10
+PALE_HORSE_PER_KILL_CRIT=4
+PALE_HORSE_MAX_CRIT=20  ; Max crit from the per-kill bonus (does not include base crit bonus)
+
+STING_RUPTURE=1  ; Rupture amount inflicted by Sting

```

Comment:
Rupture does not stack, so 1 is on the low side, and becomes worthless on plasma

Reply:
Are we thinking this is OK now? I was thinking that if you're reduced Rupture ability to 3, then 2 on Sting would be overly strong considering the holotargeting effect.

Reply:
since apparently it DOES stack with other sources, rupture 1 is fine.